internal/cmd/execute/execute.go

```diff
@@ -315,7 +316,32 @@ func (c *executeCmd) run(metadata *archive.Metadata) error {
 	switch fuzzer.Engine {
 	case "JAVA_LIBFUZZER":
 		jazzerRunner := runner.(*jazzer.Runner)
-		return jazzerRunner.ProduceJacocoReport(context.Background(), c.opts.CoverageOutputPath)
+		jacocoXMLFile, err := jazzerRunner.ProduceJacocoReport(context.Background(), c.opts.CoverageOutputPath+".xml")
+		if err != nil {
+			return err
+		}
+
+		// read the jacoco.xml report...
+		fileReader, err := os.Open(jacocoXMLFile)
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		defer fileReader.Close()
+
+		// ... and convert it to an LCOV report.
+		lcovReport, err := jacocoCoverage.ParseJacocoXMLIntoLCOVReport(fileReader)
+		if err != nil {
+			return err
+		}
+		// we don't need the jacoco.xml file anymore
+		defer os.Remove(jacocoXMLFile)
```

Comment:
```suggestion
defer func() {
    err = os.Remove(jacocoXMLFile)
		if err != nil {
			log.Errorf(err, "Failed to remove intermediate jacoco.xml report: %v", err)
    }
}()
```
Just to make sure we know if this fails. It could also be a `log.Debugf` if you prefer to keep it "hidden".

Reply:
Thx, updated!